src/main/kotlin/app/RepoExplorer.kt

```diff
@@ -3,13 +3,13 @@
 
 package app
 
-import app.utils.CommandExplore
+import app.utils.CommandAdd
 import java.io.File
 
 /**
  * RepoExplorer analyzes repositories and uploads stats to server.
  */
-class RepoExplorer(options: CommandExplore) {
+class RepoExplorer(options: CommandAdd) {
```

Comment:
If the role of CommandAdd is to interpret args from command line, we don't need RepoExplorer to depend on it. Generally, fewer depenencies is better, so you could just do 
```
class RepoExplorer(repoPath: string)
```